app/views/people/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<article>
+  <% if @all_people %>
+    <% @all_people.each do |person| %>
+      <section>
+        <div class="people-floating">
+          <strong> <h3 class="link"><%= link_to person.name, person_path(person.id) %></h3></strong>
+          <section class="people-index">
+            <strong>Uncompleted Tasks</strong> : <%= Task.where("person_id = ? AND (status = ? OR status = ?)", person.id, "Not Started", "In Progress").count %>
+          </section>
+        </div>
+      </section>
+    <% end %>
+  <% end %>
+</article>
+
+<%# person.name %>
```

Comment:
On lines 6 & 8 you used the `strong` tags. Try to do this in your css instead. It keeps all of your styles in one place which makes life easier for you (and/or whatever dev is working on the thing later on)

On lines 6 & 8 you have multiple html/erb tags. Whenever possible nest your tags. It makes the code more readable and ensures you don't have straggling brackets if you change something later.